cni/pkg/install/binaries.go

```diff
@@ -40,6 +40,18 @@ func copyBinaries(srcDir string, targetDirs []string) (sets.String, error) {
 		srcFilepath := filepath.Join(srcDir, filename)
 
 		for _, targetDir := range targetDirs {
+
+			// remove previous tmp file if some exist before creating a new one
+			// the only possible returned error is [ErrBadPattern], when pattern is malformed. Can be ignored in this case
+			matches, _ := filepath.Glob(filepath.Join(targetDir, filename) + ".tmp.*")
+			if len(matches) > 0 {
```

Comment:
Just for the sake of auditing since this is node-level, can we `installLog.Info` here saying we found stuff to clean, and are going to do so?

Reply:
sure :+1:  , updated